app/partials/desktop/confirm_modal.html

```diff
@@ -25,6 +25,15 @@
         <ng-pluralize count="count"
                      when="confirm_modal_X_messages_delete">
         </ng-pluralize>
+        <a class="tg_checkbox" ng-if="revocable" ng-click="data.revoke = !data.revoke" ng-class="data.revoke ? 'tg_checkbox_on' : ''">
```

Comment:
Hmm I'm following the Android and Desktop (macOS) clients though. The Android client behaves exactly like in this PR; the Desktop client behaves like how @stek29 commented:

> Could you please add "This will delete message just for you/for everyone" for cases when this can't be configured?
>
> (btw, seems more logical to have "unsend" on top of "update layer", not the opposite :D)

I feel the checkbox is better UI, as it represents an optional step (revoking for other users) on top of the current action (deleting for myself). If two submit buttons are used, it implies a dichotomy between "Delete for me" and "Delete for {peer_name}/ all participants" when in fact, the message is _always_ deleted for me (the user) - the question is whether or not to revoke it for the other(s).